Libraries/Animated/src/NativeAnimatedHelper.js

```diff
@@ -116,6 +116,13 @@ const API = {
     invariant(NativeAnimatedModule, 'Native animated module is not available');
     NativeAnimatedModule.disconnectAnimatedNodeFromView(nodeTag, viewTag);
   },
+  restoreDefaultValues: function(nodeTag: number): void {
+    invariant(NativeAnimatedModule, 'Native animated module is not available');
+    // Backwards compat with older native runtimes, can be removed later.
+    if (NativeAnimatedModule.restoreDefaultValues != null) {
```

Comment:
If this is running against an older runtime, does this check actually work?

Reply:
I did test running just the JS without the native changes and it worked fine. On older runtime the function doesn't exist and the disconnectFromView method handles both disconnection and restoring default values.